dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet_js_inc.jsp

```diff
@@ -601,7 +601,8 @@
         if((data["referer"] != null && data["referer"] != '' && !data["contentletLocked"]) || data["htmlPageReferer"] != null ) {
 
             if(data["isHtmlPage"]){
-                self.location = data["htmlPageReferer"];
+                var windowOrigin = window.location.protocol + "//" + window.location.hostname + (window.location.port ? ':' + window.location.port: '')
```

Comment:
this should be an util class to reuse over the app

Reply:
An util class in js?